plugins/versionpress/src/Actions/ActionsDefinitionRepository.php

```diff
@@ -31,14 +31,19 @@ public function getAllDefinitionFiles()
     public function saveDefinitionForPlugin($pluginFile)
     {
         $pluginSlug = basename(dirname($pluginFile));
-        $actionsFile = WP_PLUGIN_DIR . '/' . $pluginSlug . '/.versionpress/actions.yml';
-
-        if (!is_file($actionsFile)) {
+        $defaultActionsFile = WP_CONTENT_DIR . '/.versionpress/' . $pluginSlug . '/actions.yml';
+        $pluginDirActionsFile = WP_PLUGIN_DIR . '/' . $pluginSlug . '/.versionpress/actions.yml';
+
+        if (is_file($defaultActionsFile)) {
```

Comment:
Maybe the IF construction which is mostly repeated in all three files could be refactored to something like `$actionsFile = PluginDefinitionDiscovery::getPath($pluginSlug, 'actions.yml')`. It would also allow to avoid words like "default" and "effective" which I'm not sure are entirely fitting.